components/reddit/actions/list-comments/list-comments.mjs

```diff
@@ -0,0 +1,89 @@
+import reddit from "../../reddit.app.mjs";
+import get from "lodash/get.js";
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "action",
+  key: "reddit-action-list-comments",
+  version: "0.0.1",
+  name: "List Comments",
```

Comment:
Do you think we should name this action "List Comments in a Post" instead so its effect is clearer, and to differentiate it from potential future "List Comments <by/in X>" actions?